docs/source/about.rst

```diff
@@ -0,0 +1,20 @@
+.. _about:
+
+=====
+About
+=====
+
+StreamWebs is a dynamic networking platform that links students with locally based hands-on  watershed stewardship projects and provides a multimedia showcase for their project and data reports. StreamWebs offers teachers and community partners resources to support setting students on the path to lifelong watershed stewardship. By providing students and teachers open-source, web-based tools for watershed data management, analysis, and networking, StreamWebs supports classrooms in their pursuit of STEM (Science, Technology, Engineering, Mathematics) educational opportunities, while helping students and teachers demonstrate their role as vital contributors to watershed sustainability. StreamWebs originated in 2008 as a project of The Freshwater Trust.  In 2011, StreamWebs transitioned to Oregon State University Extension Service.
```

Comment:
The content here is good.
We tend to wrap our lines at 80 lines in `.rst` source files. Most text editors can do this automatically. If you're using vim you select the line with `shift+v` and run type `gqq` to auto-wrap lines characters (the default line wrap length is 80 characters).